### PR TITLE
feat: add helper to get command argument

### LINKED ISSUE #26: create `getCommandParameters` to unify the function to get command parameters.
**Please let us know by commenting on this issue if you want to take this issue. Thanks!**

## Task
Create a `getCommandParameters` inside the `/utils` folder to simplify and unify the function to get the parameters for a command. Currently, it scatters here:

https://github.com/teknologi-umum/bot/blob/502177ee7561d512802d97d56be44db2e34f20ee/src/services/bliding-ej.js#L8-L9
https://github.com/teknologi-umum/bot/blob/502177ee7561d512802d97d56be44db2e34f20ee/src/services/blog.js#L62-L66
https://github.com/teknologi-umum/bot/blob/502177ee7561d512802d97d56be44db2e34f20ee/src/services/covid/index.js#L14-L15
https://github.com/teknologi-umum/bot/blob/d1d915d6049a2e530512fe3d23b99d612b34d226/src/services/eval/index.js#L13-L18

## Resource
It is found that `String.substring()` is the fastest way to implement this. Try to put this [JSBin code](https://jsbin.com/selamur/edit?js,console) into [JSBench](https://jsbench.github.io/). You'll find this result:

image

<hr />

## Tugas
Buat fungsi `getCommandParameters` di dalam folder `/utils` untuk menyederhanakan & menyatukan fungsi yang digunakan untuk ambil parameter dari command. Sekarang, fungsi-fungsi ini tersebar di:

https://github.com/teknologi-umum/bot/blob/502177ee7561d512802d97d56be44db2e34f20ee/src/services/bliding-ej.js#L8-L9
https://github.com/teknologi-umum/bot/blob/502177ee7561d512802d97d56be44db2e34f20ee/src/services/blog.js#L62-L66
https://github.com/teknologi-umum/bot/blob/502177ee7561d512802d97d56be44db2e34f20ee/src/services/covid/index.js#L14-L15
https://github.com/teknologi-umum/bot/blob/d1d915d6049a2e530512fe3d23b99d612b34d226/src/services/eval/index.js#L13-L18

## Bahan pendukung
Sudah di benchmark dan ternyata `String.substring()` adalah cara tercepat dibandingkan 2 cara lain. Coba taruh [kode JSBin ini](https://jsbin.com/selamur/edit?js,console) ke [JSBench](https://jsbench.github.io/). Kamu bakal nemuin hasil ini:

image

### PR DESCRIPTION
**Pull request by elianiva**
close #26

sebenernya mau ngerjain fitur yg scraping dari ddg, tapi sekalian bikin ini dulu biar gampang dapetin querynya

btw, ini kenapa package-lock.json nya jadi punya field integrity?

### CODE REVIEW

**Comment by aldy505** [comment]
CEPET AMAT

File: src/services/eval/index.js [hunk 1]
```diff
@@ -9,16 +10,10 @@ async function evalCommand(context) {
...
-  } else if (text.startsWith(`/eval@${context.me} `)) {
-    source = text.substring(7 + context.me.length);
-  } else if (text.startsWith('```') && text.endsWith('```')) {
-    source = text.substring(3, text.length - 4);
-  } else {
-    return;
-  }
+  const source =
+    text.startsWith('```') && text.endsWith('```')
+      ? text.substring(3, text.length - 4)
```
Comment (aldy505) [inline src/services/eval/index.js:15]: ini hapus aja, ga mempan. teksnya nggak bisa di detect kayak gini.

File: src/utils/command.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Helper function to get the command arguments
+ *
+ * @param {string} cmd - Command name
+ * @param {string} message - The original message
+ * @return {string} command argument
+ */
+export const getCommandArgs = (cmd, context) => {
+  const { message, me } = context;
```
Comment (aldy505) [inline src/utils/command.js:9]: `message` here is an object. the text resides in `context.message.text`. unless you've check it somehow?

File: src/utils/command.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Helper function to get the command arguments
+ *
+ * @param {string} cmd - Command name
+ * @param {string} message - The original message
```
Comment (aldy505) [inline src/utils/command.js:5]: you specified this as a `string`, yet you supplied `context` on other files. if this is context, it should have `telegraf.Context` type.

**Review by aldy505 — CHANGES_REQUESTED** [review]
I got some notes

File: src/utils/command.js (hunk 2 above)
Comment (elianiva) [reply src/utils/command.js:9]: oh iya

File: src/utils/command.js (hunk 3 above)
Comment (elianiva) [reply src/utils/command.js:5]: WKWKW lupa belom ganti dokumentasinya

**Review by aldy505 — APPROVED** [review]
LGTM. gimana menurut @lamualfa ?

**Comment by aldy505** [comment]
ah laode lama

**Pull request merged**